src/hotspot/share/gc/shenandoah/mode/shenandoahGenerationalMode.hpp

```diff
@@ -26,6 +26,7 @@
 #define SHARE_GC_SHENANDOAH_MODE_SHENANDOAHGENERATIONALMODE_HPP
 
 #include "gc/shenandoah/mode/shenandoahMode.hpp"
+#include "oops/oop.hpp"
```

Comment:
I believe the proper way to get 'oop' into scope is to include oopsHierarchy.hpp

Reply:
Sure, will make this change.